Lucca.Logs.Netcore3.Tests/Lucca.Logs.Netcore.Tests.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

I always though you _had_ to use `TargetFramework` (singular) when only one framework was targeted 😅 

```suggestion
    <TargetFramework>net6.0</TargetFramework>
```

---

me too. that makes me want to switch to TargetFrameworks everywhere to never worry about this kind of things anymore